build-fixer.mjs

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import fs from "fs";
+const args = process.argv;
+const command = args[2];
+import { exec } from "child_process";
+
+
+if (!command) {
+  console.log("No command specified");
+  process.exit(1);
+}
+
+function deleteHelper(path) {
```

Comment:
can we rename `deleteHelper` and `copyHelper` to `deletePath` and `copyPath` for better description

Reply:
sure thing sir